de.shorsh.discord-screenaudio.yml

```diff
@@ -1,20 +1,22 @@
 app-id: de.shorsh.discord-screenaudio
 runtime: org.kde.Platform
-runtime-version: "5.15-21.08"
+runtime-version: "5.15-22.08"
 sdk: org.kde.Sdk
 base: io.qt.qtwebengine.BaseApp
-base-version: "5.15-21.08"
+base-version: "5.15-22.08"
 command: discord-screenaudio
 finish-args:
   - --share=ipc
   - --share=network
-  - --socket=x11
+  - --socket=wayland
+  - --socket=fallback-x11
   - --socket=pulseaudio
   - --filesystem=xdg-run/pipewire-0
-  - --filesystem=host
```

Comment:
Removing this would break drag-and-drop file uploading (https://github.com/maltejur/discord-screenaudio/issues/43). Do you know any fix for that?

Reply:
@maltejur Unfortunately the widget toolkits really need support for the FileTransfer protocol to properly fix this, so we have to make a compromise with the filesystem permissions :frog: 

Maybe we should use --filesystem=home or even --filesystem=xdg-(some XDG directories)

Reply:
I am not sure about that. I don't like the idea of it only working _sometimes_ (meaning when the user drags a file from their home directory). I would rather have it work for everything or nothing at all, and I would prefer that would work for everything. So I think we should keep  `--filesystem=host` for now, even it is not the ideal solution.

Reply:
@maltejur Flatpak version of Discord has more restrictive permissions though: https://thelion.website/posts/flathub-discord-permissions-explained/

Reply:
Oh, that is true. And drag and drop also doesn't work in the official discord flatpak.

![image](https://user-images.githubusercontent.com/48161361/218315148-26d21934-af50-48d8-bd74-57e601edb535.png)

Well then I would be fine with using the same permissions as the official discord app.

Reply:
@maltejur That version of Discord says this though:
> This wrapper is not verified by, affiliated with, or supported by Discord Inc.

Reply:
Ah okay. Still it would be fine by me to go with their decision to limit the filesystem access.

Reply:
@maltejur Then I'll make the filesystem permissions the same as the Discord app on Flathub :frog: